index.html

```diff
@@ -881,14 +886,29 @@ <h3>
       </li>
       <li>
         <p>
-          Otherwise, instruct the [= ICE agent =] to remove the candidate pair indicated by |candidatePair|.
+          Otherwise (if |accepted| is <code>true</code>), run the following steps:
         </p>
+        <ol>
+          <li>
+            <p>
+              [= list/Remove =] all items that [= candidate pair match | match =] |candidatePair| from |transport|.{{RTCIceTransport/[[CandidatePairs]]}}.
```

Comment:
Same here, I'd prefer identifying the candidatePair in the internal slot higher up in this algorithm where it says: 
```html
  <p>
    Let |candidatePair:RTCIceCandidatePair| be the candidate pair which is being removed.
  </p>
```
to
```html
  <p>
    [=Assert=]: the candidate pair which is being removed [=candidate pair match|matches=]
    exactly one item in |transport|.{{RTCIceTransport/[[CandidatePairs]]}}.
  </p>
</li>
<li>
  <p>
    Let |candidatePair:RTCIceCandidatePair| be the item in
    |transport|.{{RTCIceTransport/[[CandidatePairs]]}} which is being removed.
 </p>
```
After those changes, this line becomes:
```suggestion
              [= list/Remove =] |candidatePair| from |transport|.{{RTCIceTransport/[[CandidatePairs]]}}.
```